docker-compose.yml

```diff
@@ -29,12 +31,24 @@ services:
     image: mongo-express
     restart: always
     ports:
-      - 8081:8081
+      - 127.0.0.1:8081:8081
+    volumes:
+      - ./ssl/dump/mongodb-test-ca.crt:/etc/ssl/mongodb-test-ca.crt:ro
```

Comment:
Please don't use files from /ssl/dump. Requesting since name suggests not to use.
Copy this file to /ssl and then refer.